apps/angular/src/app/features/anime-dashboard/anime-dashboard.component.html

```diff
@@ -1 +1,44 @@
-<camp-anime-table [animeList]="animeList$ | async"></camp-anime-table>
+<section class="dashboard">
+	@if (isLoading$ | async){
+		<camp-basic-progress-spinner></camp-basic-progress-spinner>
+	} @else {
+		<div class="dashboard__filters">
+			<mat-form-field appearance="outline" subscriptSizing="dynamic">
+				<mat-select
+					[(ngModel)]="selectedTypes"
+					(selectionChange)="onFilter($event)"
+					placeholder="Type of an anime"
+					multiple
+				>
+					<mat-option *ngFor="let type of animeTypes" [value]="type">
+						{{ type }}
+					</mat-option>
+				</mat-select>
+			</mat-form-field>
+			<form class="search-form" (ngSubmit)="onSearch()">
+				<mat-form-field appearance="outline" subscriptSizing="dynamic">
+					<input
+						matInput
+						type="text"
+						[(ngModel)]="searchValue"
```

Comment:
```suggestion
						[(ngModel)]="searchValue"
```

Let's use `formControls`